api/src/main/java/io/opentelemetry/common/ImmutableKeyValuePairs.java

```diff
@@ -128,4 +128,22 @@ private static void swap(Object[] data, int a, int b) {
     data[b] = keyA;
     data[b + 1] = valueA;
   }
+
+  @Override
+  public String toString() {
+    final StringBuilder sb = new StringBuilder("{");
+    forEach(
+        new KeyValueConsumer<V>() {
+          @Override
+          public void consume(String key, V value) {
+            sb.append(key).append(":").append(value).append(",");
```

Comment:
```suggestion
            sb.append(key).append("=").append(value).append(", ");
```

Nit but maybe we should go ahead and match `Map`? https://github.com/openjdk/jdk/blob/master/test/jdk/java/util/AbstractMap/ToString.java#L46

Reply:
Sure. Updated, and added a test to make sure it matches.

Reply:
Of course, one downside to having this match exactly is that if you had a test that was comparing Labels and a Map, the toString would match, but of course, they will never be equal!